app-i18n/fcitx-rime/fcitx-rime-5.1.3.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg
+
+MY_PN="fcitx5-rime"
+DESCRIPTION="Chinese RIME input methods for Fcitx"
+HOMEPAGE="https://fcitx-im.org/ https://github.com/fcitx/fcitx5-rime"
+SRC_URI="https://download.fcitx-im.org/fcitx5/fcitx5-rime/fcitx5-rime-${PV}.tar.xz -> ${P}.tar.xz"
+
+LICENSE="LGPL-2.1+"
+SLOT="5"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	!app-i18n/fcitx-rime:4
+	>=app-i18n/fcitx-5.1.2:5
+	app-i18n/librime
+	app-i18n/rime-data:essay
+	virtual/libintl
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	kde-frameworks/extra-cmake-modules
```

Comment:
The slotmove ECM:5 to ECM:0  is for upcoming KDE6, maybe we should depend ECM with slot.

Reply:
There is only one slot for this package, why do we need to specify a default slot?

Reply:
As mentioned, it's for upcoming KDE6
https://github.com/gentoo/gentoo/commit/4396c608794f2d2760ef53df83426c52aef59704

Reply:
But now that there's only one solt, and there's no evidence that additional slots will be added later, is there anything wrong or improper with letting ebuild choose the default?

Reply:
Guess I got it wrong, forget about this.